### PR TITLE
Changes up a lot more messages to look like emotes.

### PR DESCRIPTION
**Pull request by NonQueueingMatt**
This changes a few very common things, like throwing and giving, so tread carefully.

### CODE REVIEW

**Comment by NonQueueingMatt** [comment]
!review

File: code/modules/mob/living/carbon/carbon.dm [hunk 1]
```diff
@@ -299,7 +299,7 @@
 				else if(istype(tapper))
 					tapper.species.tap(tapper,src)
 				else
-					M.visible_message(span("notice", "[M] taps [src] to get their attention!"), \
+					M.visible_message("<b>[M]</b> taps [src] to get their attention!"), \
```
Comment (alsoandanswer) [inline code/modules/mob/living/carbon/carbon.dm:302]:
```suggestion
					M.visible_message("<b>[M]</b> taps [src] to get their attention!", \
```

File: code/game/objects/structures/watercloset.dm [hunk 2]
```diff
@@ -467,17 +467,19 @@
...
 			if ("Empty")
 				if(!RG.reagents.total_volume)
 					to_chat(usr, span("warning", "\The [RG] is already empty."))
 					return
 
 				var/empty_amount = RG.reagents.trans_to(src, RG.amount_per_transfer_from_this)
-				user.visible_message(span("notice", "[user] empties [empty_amount]u of \the [RG] into \the [src]."),
-									 span("notice", "You empty [empty_amount]u of \the [RG] into \the [src]."))
+				var/max_reagents = RG.reagents.maximum_volume
+				//user.visible_message(span("notice", "[user] empties [empty_amount]u of \the [RG] into \the [src]."),
```
Comment (Geevies) [inline code/game/objects/structures/watercloset.dm:480]: dead code

File: code/modules/mob/living/carbon/give.dm [hunk 3]
```diff
@@ -2,9 +2,7 @@
 	set category = "IC"
 	set name = "Give"
 
-	if(incapacitated())
-		return
-	if(!istype(target) || target.stat || target.lying || target.resting || target.restrained() || target.client == null)
+	if(use_check(usr))
```
Comment (Geevies) [inline code/modules/mob/living/carbon/give.dm:5]:
```suggestion
	if(use_check(target))
```

File: code/modules/mob/living/carbon/give.dm [hunk 4]
```diff
@@ -2,9 +2,7 @@
 	set category = "IC"
 	set name = "Give"
 
-	if(incapacitated())
```
Comment (Geevies) [inline code/modules/mob/living/carbon/give.dm:5]: re-add this or usecheck this too

File: code/modules/mob/living/carbon/give.dm [hunk 5]
```diff
@@ -16,11 +14,11 @@
 		return
 
 	if(alert(target,"[usr] wants to give you \a [I]. Will you accept it?",,"Yes","No") == "No")
-		target.visible_message(span("warning", "\The [usr] tried to hand \the [I] to \the [target], \
-		but \the [target] didn't want it."))
+		target.visible_message("<b>[target]</b> pushes [usr]'s hand away.")
```
Comment (Geevies) [inline code/modules/mob/living/carbon/give.dm:17]: might seem a bit aggressive, consider something like "x doesn't take y" or "x refuses the y"

File: code/modules/mob/living/carbon/human/human.dm [hunk 6]
```diff
@@ -1952,7 +1952,7 @@
 	if(istype(heart) && !(heart.status & ORGAN_DEAD))
 		var/active_breaths = 0
 		if(!nervous_system_failure() && active_breaths)
-			visible_message("\The [src] jerks and gasps for breath!")
+			visible_message("\The <b>[src]</b> jerks and gasps for breath!")
```
Comment (Geevies) [inline code/modules/mob/living/carbon/human/human.dm:1955]:
```suggestion
			visible_message("<b>\The [src]</b> jerks and gasps for breath!")
```

File: code/modules/mob/living/carbon/give.dm (hunk 5 above)
Comment (NonQueueingMatt) [reply code/modules/mob/living/carbon/give.dm:17]: I mean, when you don't want something, you push the hand away. I don't see the aggressiveness here

File: code/modules/mob/living/carbon/give.dm (hunk 4 above)
Comment (NonQueueingMatt) [reply code/modules/mob/living/carbon/give.dm:5]: But it is use checked

**Pull request merged**